### DISCUSSION TITLE
Proxy

### DISCUSSION THREAD

**Discussion by GoodMan13iq** [pear-devs/pear-desktop#675] [Q&A]
Hello, YouTube music is not available in my region, and I want to listen to music, but in order not to turn on vpn, I decided to turn on the proxy, since there is such a function. The first time everything was fine, the input window opened, but I didn’t understand what to enter, usually when you use a proxy you need to enter the IP and port, and then some url, searching the Internet did not find useful information, and still decided to try something then write. Everything went well, but apparently the proxy did not work. Then I decided to try again, but this time the input window stopped working, just a white window and that's it. Can you explain to me how to connect a proxy to the application?
![Screenshot_2](https://user-images.githubusercontent.com/94786575/162433373-afc9f228-624d-45d7-b702-6fe46f03559d.png)

**Comment by Araxeus** [comment]
Sorry there is a bug in v1.16.0, you need to enable `in-app-menu` for that popup to work

Sorry about that, will be fixed in next version
